main.ts

```diff
@@ -399,15 +399,15 @@ class Translator {
         break;
       case ts.SyntaxKind.TemplateTail:
         this.result += `${
```

Comment:
ouch! this is a clang-format bug - it's adding the line break here.
I guess it doesn't know about backtick string literals?